### ISSUE TITLE
Sandbox is too slow to be usable

### ISSUE THREAD

**Issue by cjmyers** [drdozer/shortbol#61]
The sandbox is so slow in responding it is not usable.  I suggest not doing immediate compiles but instead include a button to compile on demand.

**Comment by drdozer** [comment]
I agree -- it is too slow, right now. The obvious reasons are a) that it is going out to the web on every keypress, and b) it isn't debursting keypresses.

**Comment by cjmyers** [comment]
Immediate updates are not useful to the user.  I suggest changing it to a compile on demand to make it usable.

Chris

**Comment by drdozer** [comment]
The real-time compilation is required to drive the editor highlighting, among other things (although this is currently not hooked in). It's also required for our number 1 user request which is to have a live diagram of the design in sync with the script. I will address the performance issues rather than disabling compilation.

**Comment by cjmyers** [comment]
In Overleaf, there is a Auto/Manual button as to when to recompile the LaTeX.  Could we have such an option?
